packages/app/src/components/pages/Home/NewFilesToReview.js

```diff
@@ -0,0 +1,70 @@
+import React, { useContext, useState } from 'react';
+import { NominationsDataContext } from '../../../utils/context/NominationsContext';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import styles from "./styles.css";
+
+const NewFilesToReview = () => {
+  const [NominationsData, setNominationsData] = useContext(NominationsDataContext);
+  const [showAll, setShowAll] = useState(false);
+  const receivedNominations = NominationsData ? NominationsData.filter(nominations => nominations.status === "received") : []
+
+  const handleClick = () => {
+    setShowAll(!showAll)
+  }
+
+  const conditionalNominationRender = () => {
+    return showAll ? receivedNominations : receivedNominations.slice(0,3)
+  }
+
+  return (
+    <div className="new-files-container">
+        <table className="home-new-files-table">
+          <thead>
+            <tr>
+              <td className="add-padding-left new-files-title">
+                <FontAwesomeIcon icon="file-image" color="green" />
+                <h1>New Files To Review</h1>
+              </td>
+              <td></td>
+              <td></td>
+              <td></td>
+              <td></td>
+              <td className="new-files-see-more">
+                <div onClick={() => handleClick()}>
```

Comment:
Since we are not passing any arguments to handleClick, do we need to wrap in in an anonymous function?   Your call, as it is a bit nit-picky.